docs/scos/dev/the-docker-sdk/202204.0/configure-services.md

```diff
@@ -454,89 +447,138 @@ BLACKFIRE_CLIENT_ID={client_id} BLACKFIRE_CLIENT_TOKEN={client-token} docker/sdk
+To enable New Relic on your local environment, a license key is required. For help with creating an API key with New Relic, refer to the following documentation: [New Relic API keys](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/)
 
-Follow the steps to enable New Relic:
+To enable New Relic for your local environment, follow these steps:
 
-1. Adjust `deploy.*.yml` in the `docker:` section:
+1. In `deploy.*.yml`, adjust the `docker` section:
 
 ```yaml
 docker:
     newrelic:
+        license: {new_relic_license}
+	distributed tracing:
+            enabled: true
 ```
 
-2. Adjust `deploy.*.yml` in the `image:` section:
+1. In the `deploy.*.yml`, adjust the `image` section:
 
 ```yaml
 image:
-    tag: spryker/php:7.4 # the image tag was previously used in `image:`
+    tag: spryker/php:7.4 # the image tag that has been previously used in `image`
     php:
         ...
         enabled-extensions:
+            ...
             - newrelic
 ```
 
-3. Pass the New Relic license:
+### Showing YVES, ZED, and GLUE as their own APM
 
-```bash
-NEWRELIC_LICENSE={new_relic_license} docker/sdk up
+By default, the APM displays in the form of `company-staging-newrelic-app` within the New Relic dashboard. This behavior can be updated so that each of the available applications shows as their own APM (for example, `YVES-DE (docker.dev)`) for improved visibility.
+
+To do so, additional changes need to be made to the Monitoring service. Within `src/Pyz/Service/Monitoring/`, you can edit `MonitoringDependencyProvider.php` to enable this functionality. You may need to add both this directory and class if they are missing from the Monitoring service.
+
+```php
+<?php declare(strict_types = 1);
+
+/**
+ * This file is part of the Spryker Commerce OS.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Pyz\Service\Monitoring;
+
+use Spryker\Service\Monitoring\MonitoringDependencyProvider as SprykerMonitoringDependencyProvider;
+use SprykerEco\Service\NewRelic\Plugin\NewRelicMonitoringExtensionPlugin;
+
+class MonitoringDependencyProvider extends SprykerMonitoringDependencyProvider
+{
+    /**
+     * @return \Spryker\Service\MonitoringExtension\Dependency\Plugin\MonitoringExtensionPluginInterface[]
+     */
+    protected function getMonitoringExtensions(): array
+    {
+        return [
+            new NewRelicMonitoringExtensionPlugin(),
+        ];
+    }
+}
 ```
 
-{% info_block warningBox "Note" %}
+With `new \SprykerEco\Service\NewRelic\Plugin\NewRelicMonitoringExtensionPlugin()` being returned within the `getMonitoringExtensions()` function, this tells the Monitoring class to include New Relic. Without these changes, New Relic only reports the base (for example, `index.php`) without showing the appropriate endpoint or class being called with each transaction.
```

Comment:
How is this related to showing apps as different APMs?

Reply:
I updated the language to this in my own PR #1331 with commit [6d2b61698a273d541ad7ae33a8e8598bc3c2a4f0](https://github.com/spryker/spryker-docs/pull/1331/commits/6d2b61698a273d541ad7ae33a8e8598bc3c2a4f0). I am unable to update this PR to show this change.